Resources/Textures/_RMC14/Interface/map_blips.rsi/meta.json

```
@@ -1,7 +1,7 @@
 {
   "version": 1,
   "license": "CC-BY-SA-3.0",
-  "copyright": "Taken from cmss13 at https://github.com/cmss13-devs/cmss13/blob/5368c7a5342a52d204eac3d6cc45228f3b976893/icons/ui_icons/map_blips.dmi, background_free by github noctyrnal, lesser_carrier by Vermidia",
+  "copyright": "Taken from cmss13 at https://github.com/cmss13-devs/cmss13/blob/5368c7a5342a52d204eac3d6cc45228f3b976893/icons/ui_icons/map_blips.dmi, background_free by github noctyrnal, lesser_carrier by Vermidia, edited runner to create lurker",
```

meta.json is fine though it should note who did it, if its you then applying this change is enough
```suggestion
  "copyright": "Taken from cmss13 at https://github.com/cmss13-devs/cmss13/blob/5368c7a5342a52d204eac3d6cc45228f3b976893/icons/ui_icons/map_blips.dmi, background_free by github noctyrnal, lesser_carrier by Vermidia, edited runner to create lurker by github DummeHytteOst",
```